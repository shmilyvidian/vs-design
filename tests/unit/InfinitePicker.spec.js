import {
  shallowMount
} from '@vue/test-utils'
import InfinitePopup from '@/packages/infinite-picker/src/index.vue'

test('render popup', () => {
  const wrapper = shallowMount(InfinitePopup)

  expect(wrapper).toMatchSnapshot()
})
