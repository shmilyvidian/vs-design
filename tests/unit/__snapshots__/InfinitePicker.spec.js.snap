// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render popup 1`] = `
<div class="infinite-picker">
  <picker-stub title="标题" showtoolbar="true" allowhtml="true" visibleitemcount="6" swipeduration="1000" defaultindex="0" columns="杭州,宁波,温州,绍兴,湖州,嘉兴,金华,衢州" toolbarposition="top" valuekey="text"></picker-stub>
</div>
`;
